sap-2-tier-marketplace-image/shared/server-Standard_multiNIC_No.json

```diff
@@ -107,30 +107,111 @@
     }
   },
   "variables": {
-    "dbDataLUNs": "0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15",
+    "dbDataLUNs": "0,1,2,3",
     "dbLogLUNs": "",
     "dbDataName": "space",
     "dbLogName": "",
-    "osProfiles": {
+    "dataDisks": [
```

Comment:
With so many disk arrays defined in the templates above, I almost think you can now use the dynamic data disks template to reduce the amount of text in these files.